src/UofIQualys/functions/public/Close-QualysSession.ps1

```diff
@@ -0,0 +1,27 @@
+<#
+.Synopsis
+   This function closes the Qualys user session created by New-QualysSession.
+.DESCRIPTION
+   This function closes the Qualys user session created by New-QualysSession.
+.EXAMPLE
+    Close-QualysSession
+#>
+function Close-QualysSession{
+    process{
+        $IVRSplat = @{
+            Headers = @{
+                "X-Requested-With"="powershell"
+            }
+            Method = 'POST'
+            URI = "$($Script:Settings.BaseURI)session/"
+            Body = @{
+                action = "logout"
+            }
+        }
+        $IVRSplat.add('WebSession',$Script:Session)
```

Comment:
Why isn't WebSession just part of declaring the splat?

Reply:
lol no idea I blame past Tam